tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -208,6 +195,7 @@ def _initialize(self, environment: Environment, variables: Dict[str, Any], lisa_
         self._runbook_name = variables["name"]
 
         self._lisa_log = lisa_log
+        self._lisa_nodes = environment.nodes
```

Comment:
The IP address used by those nodes needs to be refreshed, since LISA uses them for SSH during test cleanup.